docs/index.md

```diff
@@ -11,6 +11,20 @@ A Terraform provider for managing resources on the [Instaclustr Platform](https:
 
 It provides a flexible set of resources for provisioning and managing Instaclustr based clusters with the use of Terraform.
 
+This provider is in __General Availability__ with support for the following offerings:
```

Comment:
This looks like an error from the previous time this block was meant to be removed. Could you remove this added block from `/docs-base/index.md` and re-run the generation script?

Reply:
Team Customer Platform has been aware of this. They will fix it in PR#373 soon. I will not include `docs-base/index.md` in this commit, and remove the change in `docs/index.md`.